### PR TITLE
Hanstan/bot refactor

### PR DESCRIPTION
**Pull request by hanstananda**
* updated readme
* refactor logger
* added systemd for ami
* make some output prettier

### CODE REVIEW

**Comment by hanstananda** [comment]
Partially uses #14 2nd point

File: src/subbot/DN/STGReq.py [hunk 1]
```diff
@@ -18,27 +20,60 @@ def __init__(self, bot: commands.Bot) -> None:
...
+            data = data.loc[stg]
+            print(data)
+            entry = {
+                "Index": idx,
+                "Stat": self.stgreq_command_long[stat].title(),
+                "Cap": data[0],
+                "Penalty": data[1],
+                "% Cap": data[2]
+            }
+            res.append(entry)
```
Comment (skurnia001) [inline src/subbot/DN/STGReq.py:69]: Could be a `map` if you want.

File: src/subbot/DN/STGReq.py [hunk 2]
```diff
@@ -18,27 +20,60 @@ def __init__(self, bot: commands.Bot) -> None:
...
         else:
-            res = self.df[stat]
-            if stg:
-                res = res.loc[stg]
-            res = res.to_string()
+            stats.append(ctx.invoked_with)
+
+        # Generate and fill the output table
+        res = []
+        print(stats)
```
Comment (skurnia001) [inline src/subbot/DN/STGReq.py:53]: Delete print statement if unnecessary, or use logger instead.

**Review by skurnia001 — CHANGES_REQUESTED** [review]

File: src/subbot/DN/STGReq.py (hunk 1 above)
Comment (skurnia001) [reply src/subbot/DN/STGReq.py:69]: If don't want, please create issue instead.

Comment (hanstananda) [reply src/subbot/DN/STGReq.py:69]: Hmm isn't this a dictionary alrd? Or am I misunderstanding something?

Comment (skurnia001) [reply src/subbot/DN/STGReq.py:69]: Sorry, the `map()` function

Comment (skurnia001) [reply src/subbot/DN/STGReq.py]: nvm this

**Pull request merged**